src/Tribe/Service_Providers/Context.php

```diff
@@ -370,11 +372,11 @@ static function () {
 						],
 					],
 				],
-				'view_request'         => [
+				'view_request'       => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 9 space(s) between "'view_request'" and double arrow, but found 7.

<sub>hash: adca7fccb0fe0d26f8032f9759589ed7</sub>